### PR TITLE
Updates and cherry-pick's

### PR DESCRIPTION
**Pull request by lacksfish**
* Fix typos, fix title
* Config value LOGIN_BASED_RECOVERY for toggling cosigning or backup seed recovery
* Add variable cosign endpoint in appconfig.json

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -820,11 +847,10 @@ app.controller('walletRecoveryCtrl', function($scope, $q, $modal, $rootScope, $l
                         }
 
                         return sdk.client.post(
-                            "/wallet/" + $scope.backupDataV2.walletIdentifier + "/signbcc",
+                            "/wallet/" + $scope.backupDataV2.walletIdentifier + "/" + window.APPCONFIG.COSIGN_ENDPOINT,
```
Comment (afk11) [inline src/js/app.js:871]: hmm, when I search the diff and the appconfig.json, the value isn't set. does it need to be set for BTC/BCH?

Comment (lacksfish) [reply src/js/app.js:871]: Only if the login functionality is being used via `APPCONFIG.LOGIN_BASED_RECOVERY` set to `true`

**Comment by lacksfish** [comment]
Use with blocktrail-sdk-nodejs branch `b/walletsweeper_create_tx`

**Comment by afk11** [comment]
Updated with latest sdk

**Pull request merged**